### PR TITLE
Feature/populate owner

### PR DESCRIPTION
**Pull request by Shahar-Y**
files return with their populated owner

### CODE REVIEW

File: file/file.go [hunk 1]
```diff
@@ -79,22 +80,24 @@ const (
...
-	OwnerID     string `json:"ownerId,omitempty"`
-	Parent      string `json:"parent,omitempty"`
-	CreatedAt   int64  `json:"createdAt,omitempty"`
-	UpdatedAt   int64  `json:"updatedAt,omitempty"`
+	ID          string     `json:"id,omitempty"`
+	Name        string     `json:"name,omitempty"`
+	Type        string     `json:"type,omitempty"`
+	Size        int64      `json:"size"`
+	Description string     `json:"description,omitempty"`
+	OwnerID     string     `json:"ownerId,omitempty"`
```
Comment (omerzamir) [inline file/file.go]: should be deleted

File: file/file.go [hunk 2]
```diff
@@ -298,8 +327,20 @@ func (r *Router) GetSharedFiles(c *gin.Context) {
...
+
+		user, err := r.userClient.GetUserByID(c.Request.Context(), getUserByIDRequest)
+		if err != nil {
+			httpStatusCode := gwruntime.HTTPStatusFromCode(status.Code(err))
+			loggermiddleware.LogError(r.logger, c.AbortWithError(httpStatusCode, err))
+		}
 
-		files = append(files, createGetFileResponse(file))
+		populatedFile := createGetFileResponse(file, user.User)
+		fmt.Println(populatedFile)
```
Comment (omerzamir) [inline file/file.go]: delete

File: file/file.go (hunk 1 above)
Comment (Shahar-Y) [reply file/file.go]: done

File: file/file.go (hunk 2 above)
Comment (Shahar-Y) [reply file/file.go]: done

File: file/file.go [hunk 3]
```diff
@@ -254,7 +258,19 @@ func (r *Router) GetFileByID(c *gin.Context) {
 		return
 	}
 
-	c.JSON(http.StatusOK, CreateGetFileResponse(file, userFilePermission, foundPermission))
+	getUserByIDRequest := &uspb.GetByIDRequest{
+		Id: file.GetOwnerID(),
```
Comment (AllKinds) [inline file/file.go:262]:
```suggestion
		Id: file.GetOwnerId(),
```

**Comment by Shahar-Y** [comment]
Closing due to irrelevance to the public

**Pull request closed without merging**